src/GraphQL.Analyzers.CodeFixes/FieldBuilderCodeFixProvider.cs

```diff
@@ -236,18 +246,16 @@ static SyntaxToken GetIdentifier(SimpleNameSyntax simpleNameSyntax, string name)
                         SyntaxKind.SimpleMemberAccessExpression,
                         previousExpression,
                         IdentifierName(invocationName))
-                    .WithNewLine(newLine))
+                    .WithOperatorToken(
+                        Token(
+                            leadingTrivia,
+                            SyntaxKind.DotToken,
+                            TriviaList())))
             .WithArgumentList(
                 ArgumentList(
                         SingletonSeparatedList(
                             Argument(argumentExpression))));
 
         return exp;
     }
-
-    private static SyntaxNode Format(SyntaxNode syntaxNode, Workspace workspace)
-    {
-        syntaxNode = syntaxNode.WithAdditionalAnnotations(Formatter.Annotation);
-        return Formatter.Format(syntaxNode, Formatter.Annotation, workspace);
-    }
```

Comment:
This method was normalizing the line endings. But it also makes wrong indentations.

Reply:
Do we have a new test added for the scenario fixed by this PR?

Reply:
I have adjusted an old test. This is the result before the fix
![image](https://github.com/graphql-dotnet/graphql-dotnet/assets/4518483/dca9e81f-2fe8-4733-9203-aa262a3823c7)

Reply:
oh i see.  :+1: